orchagent/bufferorch.cpp

```diff
@@ -1161,14 +1199,16 @@ task_process_status BufferOrch::processPriorityGroup(KeyOpFieldsValuesTuple &tup
                     {
                         auto flexCounterOrch = gDirectory.get<FlexCounterOrch*>();
                         auto pgs = tokens[1];
-                        if (op == SET_COMMAND &&
+                        if (!counter_was_added && counter_needs_to_add &&
```

Comment:
@stephenxs As part of PFC detection, some platform applies zero buffer profile to queues. That profile is created only in ASIC_DB, does this counter poll bypass affect those queues as well? Can you confirm?

Reply:
@kperumalbfn 
no it doesn't 
It only bypasses the zero profiles that are applied from APPL DB.
For PFC WD, zero profiles they are applied by calling SAI API directly. In APPL DB they are still normal profiles.
So counter polling remains.